_includes/api-docs.html

```diff
@@ -8,7 +8,15 @@
 {% assign versions = api.versions | sort | where_exp: "item", "item[1] == 1" | join: "" | split: "1" | reverse %}
 ### {{ api.name }}
 {{ api.desc }}
-#### DOCS {% for version_name in versions %} {% if api.name == "libucxx" %} **[{{ version_name }} ({{ site.data.releases[version_name].ucxx_version }})](/api/{{ api.path }}/{{ version_name }})** {% else %} **[{{ version_name }} ({{ site.data.releases[version_name].version }})](/api/{{ api.path }}/{{ version_name }})** {% endif %} {% unless forloop.last %}|{% endunless %} {% endfor %}
+#### DOCS {% for version_name in versions -%}
+    {%- if api.name == "libucxx" -%}
+        **[{{ version_name }} ({{ site.data.releases[version_name].ucxx_version }})](/api/{{ api.path }}/{{ version_name }})**
+    {%- else -%}
+        **[{{ version_name }} ({{ site.data.releases[version_name].version }})](/api/{{ api.path }}/{{ version_name }})**
+    {%- endif -%}
+    {%- unless forloop.last %} | {% endunless -%}
+{%- endfor %}
```

Comment:
This indentation change was already approved, merged, and proved working in #654 ... but it needs to show up here again because that was reverted (for other, unrelated reasons) in #658